ui/src/App.js

```diff
@@ -136,12 +136,14 @@ const Admin = (props) => {
         <Resource name="playlistTrack" />,
         <Resource name="keepalive" />,
         <Player />,
-      ]}
+      ]}{' '}
```

Comment:
Can you please remove this space? Probably it was added by some reformatting by your code editor.